src/main/java/frc/robot/Robot.java

```diff
@@ -1,29 +1,38 @@
 package frc.robot;
 
 import com.chopshop166.chopshoplib.commands.CommandRobot;
+import com.chopshop166.chopshoplib.controls.ButtonXboxController;
+import com.chopshop166.chopshoplib.states.SpinDirection;
+
+import frc.robot.maps.RobotMap;
+import frc.robot.subsystems.Intake;
 
 public class Robot extends CommandRobot {
 
+  private final ButtonXboxController controller = new ButtonXboxController(0);
+
+  private final RobotMap map = getMapForName("Gaston", RobotMap.class, "frc.robot.maps");
+
+  private final Intake intake = new Intake(map.getIntakeMap());
+
   @Override
   public void robotInit() {
-    // Nothing to initialize
+    super.robotInit();
   }
 
   @Override
   public void configureButtonBindings() {
-    // TODO Auto-generated method stub
-
+    controller.a().whileHeld(parallel("Deploy and Start Roller", intake.setRollerSpeed(SpinDirection.COUNTERCLOCKWISE),
+        intake.deployIntake())).whenReleased(intake.retractIntake());
```

Comment:
Generally we create a command group inside the `intake` class that handles doing these two things so we don't have to deal with it at the robot level.

Reply:
As discussed on Wednesday, this also will get funky when you're trying to run two things in parallel from the same subsystem.